resource/labeling-app/components/utils/utils.js

```diff
@@ -81,4 +81,17 @@ export function loadConfig() {
     });
 }
 
+export function shortcut(event) {
+    const shortcuts = {
+        'multi-selection': [event.ctrlKey, event.metaKey, event.shiftKey],
+        'delete': [event.key === "Delete", event.key === "Backspace"],
+        'back': [event.code === 'ArrowLeft'],
+        'next': [event.code === 'ArrowRight', event.code === 'Enter'],
+        'skip': [event.code === 'Space']
```

Comment:
This should be `tab` and space for save&next

Reply:
Fixed in f354aad